enclave-tls/src/crypto_wrappers/openssl/gen_cert.c

```diff
@@ -0,0 +1,167 @@
+/* Copyright (c) 2020-2021 Alibaba Cloud and Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <string.h>
+#include <enclave-tls/log.h>
+#include <enclave-tls/crypto_wrapper.h>
+#include "openssl.h"
+
+static int x509_extension_add(X509 *cert, const char *oid,
+			const void *data, size_t data_len)
+{
+	int nid;
+	ASN1_OCTET_STRING *octet = NULL;
+	X509_EXTENSION *ext = NULL;
+	int ret = -1;
```

Comment:
The content is openssl so plz use SSL_SUCCESS (1). Borrow it from openssl.h

Reply:
`SSL_SUCCESS` is the return value for functions prefixed with `SSL_`, All returned here are determined 0 or 1, which is also the description of the document.

Reply:
OK no problem.